docs/content/tutorial/advanced-tutorial/resources.mdx

```diff
@@ -11,7 +11,7 @@ Jobs often interact with external resources like Hadoop/Spark clusters or data w
 
 ## Parameterizing Jobs with Resources
 
-Dagster models interactions with features of the external environment as **resources**. Dagster's library modules such as [`dagster_aws`](/\_apidocs/libraries/dagster_aws), [`dagster_gcp`](/\_apidocs/libraries/dagster_gcp), and [`dagster_slack`](/\_apidocs/libraries/dagster_slack) provide out-of-the-box implementations for many common external services.
+Dagster models interactions with features of the external environment as **resources**. Dagster's library modules such as [`dagster_aws`](/_apidocs/libraries/dagster-aws), [`dagster_gcp`](/_apidocs/libraries/dagster-gcp), and [`dagster_slack`](/_apidocs/libraries/dagster-slack) provide out-of-the-box implementations for many common external services.
```

Comment:
you might need to run `make snapshot` to pick up `/\_apidocs` format -- sorry i know it's a bit annoying. we should spend some more time improving the formatter.